defy-labs/blackmarket
# Feature/withdraw on chain

Of Note: Still needs to be styled

Description
Adds Withdraw functionality from off-chain defy

How to test
Should work with any staging account with defy.
MagicLink doesn't appear to have log in ability though.

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1204073444031489
  - https://app.asana.com/0/0/1204073444031488
